nxontology_data/efo/queries/sources.rq

```diff
@@ -0,0 +1,17 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+SELECT ?efo_id ?xref  ?axiom_source
+WHERE {
+  ?axiom rdf:type owl:Axiom.
+  ?axiom owl:annotatedSource ?source.
+  ?axiom owl:annotatedProperty oboInOwl:hasDbXref.
+  ?axiom owl:annotatedTarget ?xref.
+
+  OPTIONAL { ?axiom oboInOwl:source ?axiom_source }.
+
+  BIND( REPLACE( STR(?source), "^http.+/([^:]+)_(.+)$", "$1:$2" ) AS ?efo_id )
+}
+
+GROUP BY ?efo_id ?xref ?axiom_source
```

Comment:
Might be nice to add ORDER BY here.

Reply:
efo_id | xref | axiom_source
-- | -- | --
CHEBI:100241 | Beilstein:3568352 | Beilstein
CHEBI:100241 | CAS:85721-33-1 | ChemIDplus
CHEBI:100241 | CAS:85721-33-1 | KEGG COMPOUND
CHEBI:100241 | Drug_Central:659 | DrugCentral
CHEBI:100241 | PMID:10397494 | ChEMBL

Reply:
Ok